pkg/kubectl/cmd/label.go

```diff
@@ -287,9 +286,7 @@ func (o *LabelOptions) RunLabel() error {
 			if err != nil {
 				return err
 			}
-			if !reflect.DeepEqual(oldData, newData) {
```

Comment:
But we do want to know whether labeling did any changes, this stomps this functionality.

Reply:
Modified now. PTAL